statistics.md

```diff
@@ -0,0 +1,93 @@
+# Collecting user interactions
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Views endpoint
+**POST /api/statistics/views**
+
+To track a page view, make an HTTP POST request to this endpoint.
+
+The sections below describe the parameters for a page view request. All other information about the interaction can be derived from the request itself.
+
+### Parameters
+- `targetId`: The id of the object the user viewed
+- `targetType`: The type of the object the user viewed
+
+### Status codes:
+
+- `201` Created: if the operation succeeded
+- `422` Unprocessable Entity: if any of the parameters are missing or invalid.
+
+### Example item page view:
+
+```json
+{
+  "targetId": "43f9bb3e-f90d-458f-9858-7e4589481d18",
+  "targetType": "item"
+}
+```
+
+
+## Search Endpoint
+**POST /api/statistics/searches**
+
+To track a search, make an HTTP POST request to this endpoint.
+
+The sections below describe the parameters for a search request. All other information about the interaction can be derived from the request itself.
+
+### Parameters
+- `query`: The discovery search string.
+- `dsoType`: Limits the search to a specific DSpace Object type. Possible values:
+     - `all`
+     - `item`
+     - `community`
+     - `collection`
+- `scope`: The UUID of a specific DSpace container (site, community or collection) to which the search has to be limited.
+- `configuration`: The name of a Discovery configuration that was used by the search.
+- `appliedFilters`: An array of search filters  used to filter the result set. For more information take a look at the [search documentation](search-endpoint.md#matching-dspace-objects-search-results).
+- `page`: An object that describes the pagination status. For more information take a look at the [pagination documentation](README.md#Pagination).
+- `sort`: An object that describes the sort status. For more information take a look at the [pagination documentation](README.md#Pagination).
+
+### Status codes:
+
+- `201` Created: if the operation succeeded
+- `422` Unprocessable Entity: if any of the parameters are missing or invalid.
```

Comment:
I suggest to remove missing as a cause of the 422 because by my understanding all the property of the search document are actually optional. The only issue should be if someone specify an invalid value for some parameter such as a not existing scope, etc.

Reply:
Not all properties should be optional. Without the `query` field for example, there isn't much use in tracking the search event. But you're right in saying that the way it's written here isn't accurate. I'll try to clarify.